packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -294,7 +291,62 @@ contract L1StandardBridge is StandardBridge, Semver {
         uint32 _minGasLimit,
         bytes memory _extraData
     ) internal {
-        emit ERC20DepositInitiated(_l1Token, _l2Token, _from, _to, _amount, _extraData);
         _initiateBridgeERC20(_l1Token, _l2Token, _from, _to, _amount, _minGasLimit, _extraData);
     }
+
+    /**
+     * @inheritdoc StandardBridge
+     */
+    function _emitETHBridgeInitiated(
+        address _from,
+        address _to,
+        uint256 _amount,
+        bytes memory _extraData
+    ) internal override {
+        emit ETHDepositInitiated(_from, _to, _amount, _extraData);
+        emit ETHBridgeInitiated(_from, _to, _amount, _extraData);
```

Comment:
Instead of overriding and re-declaring the event emissions that already exist in the super function, maybe just call `super()`?

Reply:
yeah... I thought of that, but then you have to go read another function to know what it does.
If `super()` was really complicated I might prefer that, but since it's just a matter of which single line of code or the other, I'd prefer to just make that other code irrelevant.

Reply:
Got it- I see your reasoning there, makes sense 😄